apps/vaporgui/RenderHolder.cpp

```diff
@@ -224,6 +227,45 @@ void RenderHolder::_initializeNewRendererDialog(vector<string> datasetNames) {
 	}
 }
 
+void RenderHolder::_showIntelDriverWarning(const string &rendererType)
```

Comment:
What's the rational to display this warning message from RenderHolder, instead of from VolumeRenderer?

Reply:
The render library cannot display a Qt popup and we no not have a mechanism for propagating warnings